setup.py

```diff
@@ -53,16 +53,9 @@
     author_email='tvb-users@googlegroups.com',
     include_package_data=True,
     install_requires=TVB_INSTALL_REQUIREMENTS,
-    long_description="""
-This package contains the scientific library from the Virtual Brain 
-project which provides data handling and numerical routines 
-required to perform whole brain simulation. It is a work in 
-progress, and a subject of on-going research efforts. Please refer
-to the following article for more information: 
-
-http://www.frontiersin.org/Journal/10.3389/fninf.2013.00010/abstract
-
-"""
+    download_url = 'https://github.com/the-virtual-brain/tvb-library',
+    keywords = 'brain simulator neuroscience human animal neuronal dynamics models delay',
+    long_description=open('README.md').read()
```

Comment:
1. please don't be sloppy.  read the file properly, above the call to setup,
```python
with open('README.md') as fd:
    long_description = fd.read()
```
and then use the value in the call to setup
```
    long_description=long_description,
```
2. no spaces around keyword args, e.g. `a=3`, not `a = 3`

Reply:
@danvladpop thanks!